tests/test_distribution.py

```diff
@@ -84,11 +84,12 @@ def test_folder_exist(self, mock_exists):
         :meth:`~pythonforandroid.distribution.Distribution.folder_exist` is
         called once with the proper arguments."""
 
+        mock_exists.return_value = False
         self.setUp_distribution_with_bootstrap(
             Bootstrap().get_bootstrap("sdl2", self.ctx)
         )
         self.ctx.bootstrap.distribution.folder_exists()
-        mock_exists.assert_called_once_with(
+        mock_exists.assert_called_with(
```

Comment:
So is it called more than once now, how is that?
Is it worth documenting it with `call_args_list` then?

Reply:
It is called more than once, I have actually been thinking I should probably check why :joy: I changed it because it didn't seem obvious to me why it'd be important that it is only called once, it just didn't seem like a condition important to guarantee

Reply:
Yes I agree the condition may indeed not be important to guarantee. But if the behavior changed and it's not so obvious why, sometimes it's worth to dig slightly more. We may see we introduced something we didn't want to.
I can help digging later when you're done introducing the context update